addons/website_slides/views/slide_channel_views.xml

```diff
@@ -256,19 +256,29 @@
                         </t>
                         <t t-name="kanban-box">
                             <div t-attf-class="oe_kanban_color_#{kanban_getcolor(record.color.raw_value)} oe_kanban_card oe_kanban_global_click">
+                                <div class="ribbon ribbon-top-right"
+                                    attrs="{'invisible': [('active', '=', True)]}">
+                                    <field name="active" invisible="1"/>
+                                    <span class="bg-danger">Archived</span>
+                                </div>
+                                <div class="ribbon ribbon-top-right"
+                                    attrs="{'invisible': [('website_published', '!=', True)]}">
```

Comment:
Can you be published AND not active? What happens then, do the ribbons overlap each other?

Reply:
@awa-odoo  Do you mean by this `attrs="{'invisible': [('website_published', '!=', True), ('active', '=', False)]}`?

Reply:
I mean making sure that the ribbons are not overlapping each other. Modifying domains should work yes, would have to be tested.

Reply:
Agree but adding `active` domain doesn't work properly so better to keep only the published domain here. What's say?

Reply:
What do you mean "not working properly" ?

Compared to your suggestion here above, I think your missing a "|" in front of the domain.
"It should be hidden if not published OR not active". (and not AND).